core/src/main/resources/error/error-classes.json

```diff
@@ -31,10 +47,18 @@
   "CONCURRENT_QUERY" : {
     "message" : [ "Another instance of this query was just started by a concurrent session." ]
   },
+  "CONVERT_HIVE_TABLE_TO_CATALOG_TABLE" : {
```

Comment:
CONVERT_HIVE_TABLE_TO_CATALOG_TABLE does not capture the error here; CANNOT_CONVERT_HIVE_TABLE_TO_CATALOG_TABLE would better encompass it